Kernel/Syscalls/alarm.cpp

```diff
@@ -14,8 +14,8 @@ ErrorOr<FlatPtr> Process::sys$alarm(unsigned seconds)
 {
     VERIFY_NO_PROCESS_BIG_LOCK(this);
     TRY(require_promise(Pledge::stdio));
-    unsigned previous_alarm_remaining = 0;
     return m_alarm_timer.with([&](auto& timer) -> ErrorOr<FlatPtr> {
+        unsigned previous_alarm_remaining = 0;
```

Comment:
This should likely be moved into the previous commit

Reply:
Yes!